src/test/java/org/springframework/samples/petclinic/integration/StepDefinition.java

```diff
@@ -57,4 +64,67 @@ public void submitForm(String id) {
 		webElement.submit();
 	}
 
+	@Then("^I Should see all owners found$")
+	public void shouldSeeAllOwners() {
+		By elementSelector = By.xpath("//table[@id='owners']/tbody/tr/td[1]/a");
+		List<WebElement> elements = webDriver.findElements(elementSelector);
+		assertThat(elements.size()).isGreaterThan(0);
+	}
+
+	@Then("^I should see the \"([^\"]*)\" table$")
+	public void shouldSeeTable(String id) {
+		By elementSelector = By.id(id);
+		WebElement webElement = webDriver.findElement(elementSelector);
+		assertThat(webElement.isDisplayed()).isEqualTo(true);
+	}
+
+	@Then("^I should see \"([^\"]*)\" message under \"([^\"]*)\" field$")
+	public void shouldSeeMessageUnderField(String message, String fieldId) {
+		By elementSelector = By.xpath(String.format("//input[@id='%s']/parent::div/span", fieldId));
+		WebElement webElement = webDriver.findElement(elementSelector);
+		assertThat(webElement.getText()).isEqualTo(message);
+	}
+
+	@Then("^I should see field \"([^\"]*)\" with validation message:$")
+	public void shouldSeeValidationMessageUnderField(String fieldId, String message) {
+		By elementSelector = By.xpath(String.format("//input[@id='%s']/parent::div/following-sibling::span[2]", fieldId));
```

Comment:
nit.
After you're done with your work, it is a good rule of thumb to run the build of the whole project locally. If you did that, you'd see an error:
```
[ERROR]  * /Users/adam/workspace/recruitment/mariusz-rzeppa/spring-petclinic/src/test/java/org/springframework/samples/petclinic/integration/StepDefinition.java
[ERROR] 
[ERROR] Run `spring-javaformat:apply` to fix.
```
This project has very strict formatting rule applied.